src/components/GameAbout/GameAbout.module.scss

```diff
@@ -1,32 +1,119 @@
+.container {
+  display: flex;
+  width: 75rem;
+  padding: var(--space-2xl) 7.5rem;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: var(--space-lg);
+
+  @media (max-width: 768px) {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    width: 100%;
+    padding: var(--space-lg) var(--space-lg);
+  }
+}
+
 .iconOpen {
   transform: rotate(180deg);
 }
 
 .clampedText {
   display: -webkit-box;
+  -webkit-line-clamp: 3;
   -webkit-box-orient: vertical;
-  -webkit-line-clamp: 6;
   overflow: hidden;
 }
 
+.title {
+  color: var(--color-neutral-200);
+  margin: var(--space-md) 0;
+  font-size: 24px;
```

Comment:
is this font size not covered by the typography? :eyes:

Reply:
I will take a look, seems like the typography had some changes too, I will check this and come back to you specifically about if we want to change the typography now based on the new designs too.